test/index.js

```diff
@@ -1,95 +1,130 @@
-var tape = require('tape');

```

Comment:
Why all the red? Did the line endings change? If so, please use a supported [EditorConfig plugin](http://editorconfig.org/#download) for your editor to ensure this doesn't happen. The line endings should be `LF`, so let me know if the originals were screwed up and you actually fixed them here.